File: TravelAgency.Service/Scripts/core/Hoteles.js
$(document).ready(function () {
    GetAll();
});

function GetAll() {
    var item = "";
    $('#listaHoteles tbody').html('');

    $.getJSON('api/hoteles', function (data) {
        $.each(data, function (key, value) {
            item +=
                "<tr><td>" +
                value.IdHotel +
                "<tr><td>" +
                "<tr><td>" +
                value.RazonSocial +
                "<tr><td>" +
                "<tr><td>" +
                value.IdTipoDocumento
                "<tr><td>" +
                "<tr><td>" +
                value.Identificacion
                "<tr><td>" +
                "<tr><td>" +
                value.Direccion
                "<tr><td>" +
                "<tr><td>" +
                value.Telefono
                "<tr><td>" +
                "<tr><td>" +
                value.FechaCreacion
                "<tr><td>" +
                "<tr><td>" +
                value.FechaModificacion
                "<tr><td>"
        });

        $('#listaHoteles tbody').append(item);
    });
}

File: TravelAgency.Dominio.Core/Clases/Hoteles.cs
using System;
using System.Collections.Generic;
using System.Data.Common;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace TravelAgency.Dominio.Core
{
    public class Hoteles
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int IdHotel { get; set; }

        [Required]
        public string Razon_Social { get; set; }
        [Required]
        public int IdTipoIdentificacion { get; set; }
        [Required]
        public string Identificacion { get; set; }
        [Required]
        public string Direccion { get; set; }
        [Required]
        public string Telefono { get; set; }
        public DateTime FechaCreacion { get; set; }
        public DateTime FechaModificacion { get; set; }
    }
}
